_data/authors.yml

```diff
@@ -62,4 +62,7 @@ shubham_rawat:
 antonio_pedro:
   name: António Pedro
   github: antonio-pedro99
-  twitter: https://twitter.com/antdev99
\ No newline at end of file
+  twitter: https://twitter.com/antdev99
+kate_stanley:
+  name: Kate Stanley
+  github: katheris
```

Comment:
maybe you have a reason so totally fine, but you can add your twitter account here to be the link on the author name.

Reply:
I prefer not to link to Twitter, and instead tend to use GitHub as my main place to link to, although I guess if you only provide GitHub it doesn't add a link on the name right?

Reply:
Looking at the preview, it doesn't use the GitHub link. I guess it's something from the HTML template.